config/rbac/manager_role.yaml

```diff
@@ -0,0 +1,188 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: controller-manager
+  namespace: kcp-system
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - create
+      - get
+      - list
+      - patch
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - update
+      - watch
```

Comment:
Do we really need all verbs on secrets? We already had a test PR with reduced permissions where E2E tests worked: https://github.com/kyma-project/lifecycle-manager/pull/2081

Reply:
Probably this was before the introduction of istiogatewaysecret controller. We create/get/update secrets there. I guess we can omit delete from rbac role here though. 👍